### PR TITLE
Minor doc edit to add clarity around the --volume path format

### PR DESCRIPTION
**Pull request by duglin**
Also add a comment to the ValidatePath func so devs/reviewers
know exactly what its looking for.

Signed-off-by: Doug Davis dug@us.ibm.com

### CODE REVIEW

**Comment by duglin** [comment]
ping @moxiegirl @fredlf @SvenDowideit

File: docs/sources/reference/run.md [hunk 1]
```diff
@@ -1122,8 +1122,10 @@ container's `/etc/hosts` entry will be automatically updated.
 
 ## VOLUME (shared filesystems)
 
-    -v=[]: Create a bind mount with: [host-dir]:[container-dir]:[rw|ro].
-           If "container-dir" is missing, then docker creates a new volume.
+    -v=[]: Create a bind mount with: [host-dir:]container-dir[:rw|ro].
```
Comment (moxiegirl) [inline docs/sources/reference/run.md]: Tics not quotes please.  Sorry, I see this was let through before and you were just following a bad precedent.

Comment (duglin) [inline docs/sources/reference/run.md]: yep - just following someone else's lead :-)  fixed.

Comment (thaJeztah) [inline docs/sources/reference/run.md]: Given that this approach (ie, a code-block describing the flag/option) is already used extensively in this document, I think this is fine (for consistency).

Comment (thaJeztah) [inline docs/sources/reference/run.md]: @duglin LOL

**Comment by thaJeztah** [comment]
LGTM, thanks @duglin

**Comment by moxiegirl** [comment]
LGTM

**Comment by moxiegirl** [comment]
dank @duglin

**Comment by thaJeztah** [comment]
Oh, i18n thanking! :)

Dankjewel, @duglin

**Comment by moxiegirl** [comment]
@jfrazelle cherry pick on this one please

**Comment by jfrazelle** [comment]
cherry-pciked

**Pull request merged**